cmd/lakefs/cmd/migrate.go

```diff
@@ -18,32 +20,49 @@ var versionCmd = &cobra.Command{
 	Use:   "version",
 	Short: "Print current migration version and available version",
 	Run: func(cmd *cobra.Command, args []string) {
-		logger := logging.Default()
 		cfg := loadConfig()
-
-		ctx := cmd.Context()
 		kvParams := cfg.GetKVParams()
+		ctx := cmd.Context()
 		kvStore, err := kv.Open(ctx, kvParams)
 		if err != nil {
-			logger.WithError(err).Fatal("Failed to open KV store")
+			_, _ = fmt.Fprintf(os.Stderr, "Failed to open KV store: %s\n", err)
```

Comment:
Isn't it the same as log.Fatal?

Reply:
Removed the use of logging for this command.
The logger fatal will print out to the log + exit(1).
The Fprintf will write to std err without and added a call to os.Exit after.
The log output format and file/stderr is based on the lakefs configuration, while the Fprint is not.